trpc/queries/goals.ts

```diff
@@ -510,7 +510,7 @@ export const addCalclulatedGoalsFields = (goal: any, activityId: string) => {
     }
     checkParent(goal.project);
 
-    const _isEditable = _isOwner || _isIssuer || parentOwner;
+    const _isEditable = _isOwner || _isIssuer || parentOwner || role === 'ADMIN';
```

Comment:
can't we add just isAdmin to activity as a calculated field?

Reply:
we can, but we don't have special actions for admins only currently